### PR TITLE
Add MMF aquaplanet compsets

### PR DESCRIPTION
**Pull request by whannah1**
Adds compset definition and use case file for running 1-mom MMF in aquaplanet mode. 

[BFB]

### CODE REVIEW

File: components/cam/cime_config/config_compsets.xml [hunk 1]
```diff
@@ -353,6 +353,52 @@
     <lname>2000_CAM5%AQUA_SLND_SICE_DOCN%AQPFILE_SROF_SGLC_SWAV</lname>
   </compset>
 
+  <!-- MMF aquaplanet compsets -->
+  <compset>
+    <alias>F-MMF1-AQP1</alias>
+    <lname>2000_CAM5%MMF1-AQUA_SLND_SICE_DOCN%AQP1_SROF_SGLC_SWAV</lname>
```
Comment (lee1046) [inline components/cam/cime_config/config_compsets.xml:359]: Did you set the values related with AQP[1-10] in config_components.f90?

Comment (whannah1) [reply components/cam/cime_config/config_compsets.xml:359]: yes, the "DOCN" part is what sets the SST pattern associated with the number.

**Comment by brhillman** [comment]
Merged to next.

**Comment by brhillman** [comment]
Merged to master.

**Pull request merged**